VS2012.TestAdapter/ChutzpahAdapterSettings.cs

```diff
@@ -38,6 +38,11 @@ public ChutzpahAdapterSettings() : base(AdapterConstants.SettingsName)
         /// </summary>
         public bool OpenInBrowser { get; set; }
 
+        /// <summary>
+        /// The arguments for the corresponding browser if running in browser.
+        /// </summary>
+        public string BrowserArgs { get; set; }

```

Comment:
I am moving away from adding settings in the VS dialogs. Please add to the chutzpah.json file instead.